### DISCUSSION TITLE
Inlinekeyboardmarkup gives 'str' object has no attribute 'items'

### DISCUSSION THREAD

**Discussion by edguru** [aiogram/aiogram#1288] [General]
steps to reproduce
use the following keyboard
buttons = InlineKeyboardMarkup(inline_keyboard=[[InlineKeyboardButton(text= "Play Flappy Bird", callback_game="flappy_bird"),
        InlineKeyboardButton(text ="Play with Friends", switch_inline_query_chosen_chat="" ),
        ]])
This raises the error 'str' object has no attribute 'items'
this looks like an issue with the json parser

**Comment by JrooTJunior** [comment]
Fixed by #1290
